design/scenarios.md

```diff
@@ -0,0 +1,73 @@
+# Working Scenarios of Kata Container
+
+- [CRI](#cri)
+- [OCI based CRI Runtimes](#oci-based-cri-runtimes)
+  - [PodSandbox in OCI based runtimes](#podsandbox-in-oci-based-runtimes)
+  - [Typical working pattern of CRI runtimes](#typical-working-pattern-of-cri-runtimes)
+- [Frakti CRI Runtime](#frakti-cri-runtime)
+  - [The APIs Frakti rely on](#the-apis-frakti-rely-on)
+  - [Working pattern of Frakti](#working-pattern-of-frakti)
+- [Working with Docker](#working-with-docker)
+
+## CRI
+
+The operations defined in CRI RuntimeService are quite straightforward. There are two parts (as shown in the figure): PodSandbox operations and Container Operations.
+
+![The CRI RuntimeService](cri-runtime-service.png)
```

Comment:
Please could you add the sources for the images on this PR (ideally in SVG format, but atleast in an open format) to make it easier to update the images on a later PR without having to redraw?

Reply:
agreed.